draft-irtf-cfrg-vdaf.md

```diff
@@ -3885,15 +3885,15 @@ def prep_init(Poplar1, verify_key, agg_id, agg_param,
         corr_xof = Poplar1.Xof(
             corr_seed,
             Poplar1.domain_separation_tag(USAGE_CORR_INNER),
-            byte(agg_id) + nonce,
+            byte(agg_id) + nonce + to_be_bytes(self.threshold),
```

Comment:
`prep_init()` is a classmethod; the class is called `Poplar1` in this scope.

(Sorry, in retrospect, class factories was probably the wrong paradigm for this project. cc/ @divergentdave.)